src/config/scrivitoSites.ts

```diff
@@ -58,7 +58,9 @@ export function siteForUrl(
 }
 
 function languageForUrl(url: string) {
-  const regex = new RegExp(`^${getBaseAppUrl()}\\/(?<lang>[a-z]{2})([?/]|$)`)
+  const regex = new RegExp(
+    `^${getBaseAppUrl()}\\/(?<lang>[a-z]{2}(-[A-Z]{2})?)([?/]|$)`,
```

Comment:
According to [RFC 5646](https://www.ietf.org/rfc/bcp/bcp47.txt) in "2.1.1.  Formatting of Language Tags":

> At all times, language tags and their subtags, including private use and extensions, are to be treated as case insensitive: there exist conventions for the capitalization of some of the subtags, but these MUST NOT be taken to carry meaning.
>
> Thus, the tag "mn-Cyrl-MN" is not distinct from "MN-cYRL-mn" or "mN-cYrL-Mn" (or any other combination), and each of these variations conveys the same meaning: Mongolian written in the Cyrillic script as used in Mongolia.

So `lang` should be treaded `cAsE iNsensitIVE`.

Reply:
Since we only handle two-letter languages so far, and the UI automatically uppercases the locale part, I went with the simple implementation.
Still update?

Reply:
Nice. I didn't know, that the SDK automatically corrects the casing.